examples/set-gen/sets/set_test.go

```diff
@@ -118,6 +118,25 @@ func TestStringSetDifference(t *testing.T) {
 	}
 }
 
+func TestStringSetSymmetricDifference(t *testing.T) {
+	a := NewString("1", "2", "3")
+	b := NewString("1", "2", "4", "5")
+	c := a.SymmetricDifference(b)
+	d := b.SymmetricDifference(a)
+	if len(c) != 3 {
+		t.Errorf("Expected len=3: %d", len(c))
+	}
+	if !c.Equal(NewString("3", "4", "5")) {
```

Comment:
Nit: equal already checks the length for you, you don't need to check it separately.